File: 0062-unique-paths/0062-unique-paths.cpp
class Solution {
public:
    int uniquePaths(int m, int n) {
        vector<vector<int>> dp(m, vector<int>(n, 1));
        for(int i=m-2;i>=0;i--){
            for(int j=n-2;j>=0;j--){
                dp[i][j] = dp[i+1][j] + dp[i][j+1];
            }
        }
        
        return dp[0][0];
    }
};

File: 0062-unique-paths/0062-unique-paths.ts
function uniquePaths(m: number, n: number): number {
    let dp = Array.from(Array(m), ()=>new Array(n).fill(1));
    for(let i=1;i<m;i++){
        for(let j=1;j<n;j++){
            dp[i][j]= dp[i-1][j]+dp[i][j-1];
        }
    }
    
    return dp[m-1][n-1];
};